packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -888,8 +888,30 @@ export default class Meeting extends StatelessWebexPlugin {
         this.updateShare({
           sendShare: false,
           receiveShare: this.mediaProperties.mediaDirection.receiveShare
+        }).then(() => {
+          Trigger.trigger(
+            this,
+            {
+              file: 'meetings/index',
+              function: 'remoteShare'
+            },
+            EVENT_TRIGGERS.MEETING_STARTED_SHARING_REMOTE
+          );
```

Comment:
This only works if the current user had the share before someone else took it. You want to do the opposite conditional of line 903-904 to trigger this.

Reply:
@mwegman does it look ok now ? or we are still missing the condition you mentioned ?